src/main/java/edu/mit/ll/sql/connection/sql.properties

```diff
@@ -0,0 +1,4 @@
+username=root
+password=secret
+driver=org.mysql.jdbc.Driver
+address=jdbc:mysql://somehost:5432/someDb
```

Comment:
Hopefully it is documented somewhere to change these configs for the user's environment?

Reply:
Doesn't look like it is. I'm looking more into the workflow and where it's used to try to find the best place to document it. I would think anything on the D4M side would have its own place to specify these properties.

Reply:
Turns out none of the files in this directory are currently being used. They've been removed.